src/api.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
```

Comment:
Nothing wrong with this file at all! But you could also create a global axios instance with the base url baked into it like so:

```javascript
import axios from 'axios';
import config from '@/config';

const http = axios.create({
  baseURL: config.BASE_URL
});

export { http };
```
That way it can be imported into another file and calls can be made like so:

```javascript
await http({
    // axios options
});
```
BUT AGAIN! Nothing wrong with the way you did it! Just presenting another option.

Reply:
Also, this is assuming you have a config file at the root of the project 😄